### ISSUE TITLE
Removing Essentia (wait, what?)

### ISSUE THREAD

**Issue by Fortanono** [Azanor/thaumcraft-suggestions#8]
Yes, I really am suggesting this. TC5 is really just TC4 in new clothing, and I had an idea to shake things up a bit more and really add some more thought into the mod. That would be taking the new aura system a step further and replacing essentia with vis entirely. 

The idea would be that you could combine different types of vis with a machine by putting a "catalyst" item in to determine what aspects will be added to the aura (depending on what aspects the item has, and it'd still get consumed), along with the vis types that would combine to make that aspect. Vis would be consumed by Infusion, the Crucible and much else.

For example, if I wanted to make Lux vis, I could put Torches in to combine one unit of Ignis and one unit of Aer, but they'd go by quickly, or you could use Glowstone and it would last longer, but also combine Aer and any Spiritus into Sensus. In addition, if there is no Spiritus to speak of, Vitium would go into the aura instead (which would no longer be a true "aspect", as no items would contain it).

**Comment by carso150** [comment]
I think that azanor already found a mechanism for vis and esencia that works, the old ones (thaumcraft 1 and 2) are only test and thaumcraft 3 is the begining of the end of the big changes, this world really well, maybe it need upgrades, but not scratch everything

**Comment by Fortanono** [comment]
I just find it weird that aspects exist in two forms, and one only has primal versions. This system would also really get you to think about how to produce certain aspects, what items won't do much damage, etc.

This _would_ warrant a TC6 if used, but let's see if Azanor likes it.

**Comment by Argis13** [comment]
Actually, nodes can have compound aspects, it's just rare and you get all the aspects that make that up if you try to draw from it.

**Comment by Azanor** [comment]
Yeah, the whole aspect system is an essential part of TC now. It does have problems and in some ways to pre-aspect TC mechanics where a lot simpler to handle and understand. Removing aspects now however would probably alienate a lot of fans and many appreciate the complexity of the new system.

**Comment by Fortanono** [comment]
Makes sense. Keep in mind though, that this my proposed system still requires you to distill items, but it adds another layer of complexity due to having to combine the already-existing types of Vis. I feel like this system is more complex if anything, and it feels more magical than the previous one.

But whatever, you created the mod. I'm at least glad you responded to such a big idea. Just wanted to clear some stuff up.
